parser/src/lexer/tests.rs

```diff
@@ -45,11 +45,16 @@ fn keywords_with_identifiers() {
 
 #[test]
 fn multi_arithmetic_ops() {
-    let source = "enf clk' - clk - 1 = 0";
+    let source = "enf clk'^2 - clk' - clk - 1 = 0";
```

Comment:
Same comment here that I left in the `Mul` PR about modifying this test vs. adding new tests